mypyc/ir/pprint.py

```diff
@@ -222,15 +222,16 @@ def visit_call_c(self, op: CallC) -> str:
     def visit_primitive_op(self, op: PrimitiveOp) -> str:
         args = []
         arg_index = 0
-        type_arg_index = 0
+        # type_arg_index = 0
         for arg_type in zip(op.desc.arg_types):
             if arg_type:
                 args.append(self.format("%r", op.args[arg_index]))
                 arg_index += 1
             else:
-                assert op.type_args
-                args.append(self.format("%r", op.type_args[type_arg_index]))
-                type_arg_index += 1
+                assert False
+                # assert op.type_args
+                # args.append(self.format("%r", op.type_args[type_arg_index]))
+                # type_arg_index += 1
```

Comment:
What's this change?

Reply:
I'm not sure -- it seems this code is somehow unreachable (there's no `type_args` attribute) but isn't marked as such. I replaced it with an `assert False` because I'm not sure about it being unreachable and I wanted to check against the test cases.

Reply:
I'm going to assume the code is unreachable and add an assert ensuring as much.

Reply:
It absolutely is, somehow I missed that we're checking `if <single element tuple>` which... always true, why would this ever be false. I'm going to assume that since it's unnecessary to pass test cases, it's unnecessary to keep.